File: src/main/java/com/lessmarkup/interfaces/recordmodel/InputFile.java
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */
package com.lessmarkup.interfaces.recordmodel;

public class InputFile {
    private String type;
    private byte[] file;
    private String name;

    public String getType() {
        return type;
    }
    public void setType(String type) {
        this.type = type;
    }

    public byte[] getFile() {
        return file;
    }
    public void setFile(byte[] file) {
        this.file = file;
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
}

File: src/main/scala/com/lessmarkup/interfaces/recordmodel/InputFile.scala
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

package com.lessmarkup.interfaces.recordmodel

class InputFile {
  private var `type`: String = null
  private var file: Array[Byte] = null
  private var name: String = null

  def getType: String = `type`

  def setType(`type`: String) {
    this.`type` = `type`
  }

  def getFile: Array[Byte] = file

  def setFile(file: Array[Byte]) {
    this.file = file
  }

  def getName: String = name

  def setName(name: String) {
    this.name = name
  }
}